### PR TITLE
Tutorial polling

### PR DESCRIPTION
**Pull request by amn41**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: docs/_static/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,258 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```
Comment (amn41) [inline docs/_static/css/custom.css]: can we move this to our fork of the theme? 

https://github.com/RasaHQ/sphinx_rtd_theme

@tmbo

Comment (tmbo) [reply docs/_static/css/custom.css]: sure :+1:

Comment (amn41) [reply docs/_static/css/custom.css]: @psharmz gave you write permission to https://github.com/RasaHQ/sphinx_rtd_theme

**Comment by amn41** [comment]
@tmbo could you please have a look? @psharmz please sign the CLA :)

File: docs/_templates/layout.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% extends "!layout.html" %}
 
-{% set css_files = css_files + ['_static/custom.css'] %}
+{% set css_files = css_files + ['_static/css/custom.css'] %}
```
Comment (tmbo) [inline docs/_templates/layout.html]: this change breaks the css for me. why do you need it?

Comment (psharmz) [reply docs/_templates/layout.html]: you you're right you actually dont need it - reverting

**Comment by tmbo** [comment]
@psharmz please let me know when you are ready so I can take another look at it.

File: docs/_templates/layout.html (hunk 2 above)
Comment (psharmz) [reply docs/_templates/layout.html]: @tmbo done.

File: docs/conf.py [hunk 3]
```diff
@@ -151,7 +151,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = ['_static/css/custom.css']
```
Comment (tmbo) [inline docs/conf.py]: is this necessary? I think this will prevent any images from getting loaded.

**Comment by psharmz** [comment]
@amn41 @tmbo i think i've fixed everything- can you give this one last look and lmk if you think its ready for merge?

**Pull request merged**